src/ui/src/builder/settings/BuilderTemplateInput.vue

```diff
@@ -101,10 +104,10 @@ const props = defineProps({
 	inputId: { type: String, required: false, default: undefined },
 	value: { type: String, required: false, default: undefined },
 	multiline: { type: Boolean, required: false },
-	variant: {
-		type: String as PropType<"code" | "text">,
+	rows: {
```

Comment:
since `rows` only makes sens only when it's `multiline`, I would prefix it like
```suggestion
	multilineRows: {
```